### ISSUE TITLE
[2|CBH & HR - Nav Bar] Navbar Linkage

### ISSUE THREAD

**Issue by hzz4343** [Builder-s-League/BuildersLeague-Edition1#84]
<h2 align="center"> 🍰 Appetite 🍰 </h2>
<div align="center">

| Property | Value |
|:-------|:------:|
| Points | 15 points |
| Difficulty rating | ★★★☆☆| 
| Maximum Assignees | 3 person | 

</div>

🤔 Problem Statement
------------------------------------------
Realize linkage of each dashboard.

🧪 Required Tests
-------------------------------------
Mainly screenshot tests, and some integration tests for Supabase and Payload CMS.
_Review the [Code & Testing Guidelines](https://google.com) for more info._

⚠️ Careful about:
-------------------
No major rabbit holes identified. But always be careful with scope creep!

🤖 Technologies focused on in this feature
---------------------------
- React
- Tailwind
- Shadcn
- NextJS
- Supabase/PostGresQL
- AppFlowy

**Comment by calmHeaven** [comment]
I want to work on this @al-Sajjadieh

**Comment by hzz4343** [comment]
@calmHeaven Sorry, this one is temporary closed. Please work on another one (from feature #77 23 9 or 3)

**Comment by NadiiaTysh** [comment]
is it unlocked?

**Comment by hzz4343** [comment]
@NadiiaTysh  Yes Naddia, it's unlocked now. Let me know if you want to be assigned

**Comment by PJ-Bernal** [comment]
Im going to work on this one as well.

**Comment by NadiiaTysh** [comment]
@hzz4343  I'd like it, thank you

**Comment by josedsegurav** [comment]
Is this still open?

**Comment by PJ-Bernal** [comment]
No, the whole feature was done already.
